sdk/ai/azure-ai-inference/README.md

```diff
@@ -535,15 +535,44 @@ For more information, see [Configure logging in the Azure libraries for Python](
 
 To report issues with the client library, or request additional features, please open a GitHub issue [here](https://github.com/Azure/azure-sdk-for-python/issues)
 
-## Tracing
+## Observability With OpenTelemetry
+
+The Azure AI Inference client library provides experimental support for tracing with OpenTelemetry.
+
+You can capture prompt and completion contents by setting `AZURE_TRACING_GEN_AI_CONTENT_RECORDING_ENABLED` environment to `true` (case insensitive).
+By default prompts, completions, function name, parameters or outputs are not recorded.
 
-The Azure AI Inferencing API Tracing library provides tracing for Azure AI Inference client library for Python. Refer to Installation chapter above for installation instructions.
+### Setup with Azure Monitor
 
-### Setup
+When using Azure AI Inference library with [Azure Monitor OpenTelemetry Distro](https://learn.microsoft.com/azure/azure-monitor/app/opentelemetry-enable?tabs=python),
+distributed tracing for Azure AI Inference calls is enabled by default when using latest version of the distro.
 
```

Comment:
Instead of "Distro" and "distro" can we use the full word? I think cspell will error on this.

Reply:
"Distro" is the actual name of the product and it is how Azure monitor advertises it. https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/monitor/azure-monitor-opentelemetry#azure-monitor-opentelemetry-distro-client-library-for-python